pulp_ansible/app/viewsets.py

```diff
@@ -129,9 +134,25 @@ def filter_by_q(self, queryset, name, value):
         )
         return qs.annotate(rank=ts_rank_fn).order_by("-rank")
 
+    def filter_by_tags(self, qs, name, value):
+        """
+        Filter queryset qs by list of tags.
+
+        Args:
+            qs (django.db.models.query.QuerySet): CollectionVersion queryset
+            value (string): A comma separated list of tags
+
+        Returns:
+            Queryset of CollectionVersion that matches all tags
+
+        """
+        for tag in value.split(","):
+            qs = qs.filter(tags__name=tag)
+        return qs
```

Comment:
I think we can get rid of the loop, and do only one query, changing to:
```python
qs = qs.filter(tags__name__in=value.split(","))
```

Reply:
They have slightly different behavior:

```
In [1]: CollectionVersion.objects.filter(tags__name="nada").filter(tags__name="k8s")                                                         
Out[1]: <QuerySet []>                                                                                                                       

In [2]: CollectionVersion.objects.filter(tags__name__in=["nada", "k8s"])
Out[2]: <QuerySet [<CollectionVersion: k8s>, <CollectionVersion: k8s_demo_collection>]>
```

We want the first behavior. I will add a test for this.

Reply:
Cool! Like `or` and `and`